src/js/pinny.js

```diff
@@ -102,6 +104,8 @@
                 // open and locked the viewport up already
                 !this._activePinnies() && this.$pinny.lockup('lock');
 
+                SyntheticResize.startWatching();
```

Comment:
Correct me if I'm wrong, but I thought this was supposed to only monitor if it was iOS 7. Wouldn't we do that check here, and invoke a callback if it fired? Also, didn't we decide we would call this function `on`, and pass in the resize event and callback?